pfcpiface/conn.go

```diff
@@ -134,6 +152,7 @@ func (node *PFCPNode) NewPFCPConn(lAddr, rAddr string, buf []byte) *PFCPConn {
 		InstrumentPFCP: node.metrics,
 		hbReset:        make(chan struct{}, 100),
 		hbCtxCancel:    nil,
+		pConnID:        node.getConnId(),
```

Comment:
I believe we should come up with a better allocation strategy for `connId`s. Refer to https://github.com/omec-project/upf-epc/blob/master/pfcpiface/up4.go#L71 as an example.